cinema/views.py

```diff
@@ -0,0 +1,41 @@
+from django.shortcuts import get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+
+from .models import Movie
+from .serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])

```

Comment:
Consider using class-based views (e.g., `ListCreateAPIView` and `RetrieveUpdateDestroyAPIView`) from Django REST Framework for better organization and reusability of code. They provide built-in methods for common operations and can simplify your view logic.